content/en/account_management/billing/pricing.md

```diff
@@ -64,6 +64,13 @@ Datadog has many pricing plans to fit your needs. For more information, see the
   * Each host is allowed up to 4 profiled containers for free. Overage containers will be priced at $2 per container. Note: this allotment is aggregated across all hosts so if you have 4 containers on average across all your hosts, you will not be charged overages on host-by-host basis.
 * Datadog measures the total number of containers that are being profiled. A container is a self-contained operating environment that includes application software and limited operating system libraries and settings. Once every five minutes, Datadog records the number of unique containers you are monitoring in the Datadog Continuous Profiler service. Datadog charges monthly based on the fractional hours of monitored containers. For Continuous Profiler, Datadog only counts the containers that are running the Continuous Profiler service towards the total monitored container count.
 
+## Incident Management 
+
+* Datadog tracks the number of monthly active users who participate in incident management and response. 
```

Comment:
Datadog tracks the number of unique active users per month, who participate in incident management and response.

Reply:
FYI @marcweisman